### ISSUE TITLE
Output redline in BBCode style

### ISSUE THREAD

**Issue by houfu** [houfu/redlines#39]
Currently, redlines output in markdown, including GHFM flavors. (See [docs](https://houfu.github.io/redlines/redlines/redlines.html#Redlines.output_markdown))

It would be good to output in [BBCode](https://en.wikipedia.org/wiki/BBCode) format as a distinct style in output_markdown.

- [ ] Add style to `output_markdown`
- [ ] Add test
- [ ] Add docs

**Comment by tharun634** [comment]
Hey, I'd like to work on this.

Just so I understand, is this the expectation:

```python
from redlines import Redlines

test = Redlines(
    "The quick brown fox jumps over the lazy dog.",
    "The quick brown fox walks past the lazy dog.",
    markdown_style="bb-code"  # This option specifies the style as bb-code
)

test.output_markdown() # The quick brown fox [s]jumps over [/s][u]walks past [/u]the lazy dog.
```

**Comment by houfu** [comment]
Yup, though
* it should be "bbcode"
* It would be better if it was colored red for deletion and green for addition.

**Comment by houfu** [comment]
Merged in 0.5
